grails-app/domain/org/pih/warehouse/order/OrderAdjustment.groovy

```diff
@@ -104,18 +106,34 @@ class OrderAdjustment implements Serializable, Comparable<OrderAdjustment> {
         return !postedPurchaseInvoiceItems.empty
     }
 
+    HashSet<InvoiceItem> getInvoicedItems() {
```

Comment:
`Set` should be the type here. We should use interfaces as types instead of a particular implementation of that interface, which is `HashSet` in this case.